content/developer/reference/backend/orm.rst

```diff
@@ -1063,13 +1063,15 @@ Map
 
 .. note::
 
-    Since V13, multi-relational field access is supported and works like a mapped call:
+    Since V13, multi-relational field access is supported and works like the old mapped call.
+    Since V19, mapped returns a list where the path or function is applied to each element in self.
```

Comment:
In the documentation, we try not to mention version changes. The content of the doc in a given version should only describe the behavior of the framework in that version. Here, in particular, I think that "since V13, [...] works like the **old** mapped call" will confuse the reader more than it will help them.